docs/docs/development/api.md

```diff
@@ -1285,3 +1287,9 @@ For example, the application may be able to register a URL that BigBlueButton wo
 - http&#58;//third-party-app/bbb-integ.php?event=meetingEnded&meetingID=abcd
 - http&#58;//third-party-app/bbb-integ.php?event=userLeft&userID=1234
 - http&#58;//third-party-app/bbb-integ.php?event=meetingEnded&meetingID=abcd
+
+## Other Topics
+
+### Presentation Formats
+
+See valid formats for uploading a presentation in the filter list of [back-end](https://github.com/bigbluebutton/bigbluebutton/blob/6333cc42ffbe9bae0840db68a20404bc3e7b78df/bbb-common-web/src/main/java/org/bigbluebutton/presentation/MimeTypeUtils.java#L28) and of [front-end](https://github.com/bigbluebutton/bigbluebutton/blob/6333cc42ffbe9bae0840db68a20404bc3e7b78df/bigbluebutton-html5/private/config/settings.yml#L939).
```

Comment:
It might be better to use a tag rather than the commit ( would make updating this easier in the future ).
I notice we're making the change on BBB 3.0, so likely we can backport it to 2.7 and use 2.7.10 as the tag for the links.

Reply:
I updated to 2.7.10, it is not there yet, but it is already prepared for next release.